src/Multiselect/MultiselectComponents.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import classNames from 'classnames';
+import { components } from 'react-select';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import { Close, KeyboardArrowDown } from '../../icons';
+
+const ClearIndicator = (props) => (
+  <components.ClearIndicator {...props}>
+    <Icon src={Close} />
+  </components.ClearIndicator>
+);
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <Icon src={KeyboardArrowDown} />
```

Comment:
Similar question regarding `IconButton` vs `Icon`

Reply:
The same here